packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -34,10 +35,13 @@ type BaseMemberType =
 
 type MemberType = BaseMemberType | BaseMemberType[];
 
-type Order =
+type AlphabeticalOrder =
   | 'alphabetically'
   | 'alphabetically-case-insensitive'
-  | 'as-written';
+  | 'natural'
+  | 'natural-case-insensitive';
```

Comment:
Did we want to make this an enum given we're using the string values directly?

Reply:
Yeah. I added a note to #5392 to do this holistically.